### DISCUSSION TITLE
数据库版本如何针对不同用户开放使用不同的模型

### DISCUSSION THREAD

**Discussion by hal0n0sec** [lobehub/lobehub#4582] [General Question]
如题所示，是否可以实现针对不同的用户，例如用户A可以使用gpt4o和claude，用户B只能使用claude，用户C只能使用gpt4o。
如果可以的话，请问具体该如何进行配置？

**Comment by SpeedupMaster** [comment]
这个暂时不行，只能通过给不同用户分发不同的API KEY来控制。

**Reply by hal0n0sec** [reply]
好的，我明白了

**Comment by arvinxx** [comment]
这不属于开源版会提供的功能
